src/main/java/com/conveyal/datatools/manager/jobs/validation/SharedStopsValidator.java

```diff
@@ -0,0 +1,110 @@
+package com.conveyal.datatools.manager.jobs.validation;
+
+import com.conveyal.datatools.manager.models.Project;
+import com.conveyal.gtfs.error.NewGTFSError;
+import com.conveyal.gtfs.error.NewGTFSErrorType;
+import com.conveyal.gtfs.error.SQLErrorStorage;
+import com.conveyal.gtfs.loader.Feed;
+import com.conveyal.gtfs.model.Stop;
+import com.conveyal.gtfs.validator.FeedValidator;
+import com.csvreader.CsvReader;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+
+public class SharedStopsValidator extends FeedValidator {
+    Feed feed;
+    Project project;
+
+    public SharedStopsValidator(Project project) {
+        super(null, null);
+        this.project = project;
+    }
+
+    // This method can only be run on a SharedStopsValidator that has been set up with a project only
+    public SharedStopsValidator buildSharedStopsValidator(Feed feed, SQLErrorStorage errorStorage) {
+        if (this.project == null) {
+            throw new RuntimeException("buildSharedStopsValidator can only be called with a project already set!");
```

Comment:
I think this would read better if it said "Shared stops validator can not be called because no project has been set."